packages/core/src/external/carequality/ihe-gateway-v2/outbound/xcpd/create/iti55-envelope.ts

```diff
@@ -25,7 +25,7 @@ export type BulkSignedXCPD = {
   outboundRequest: OutboundPatientDiscoveryReq;
 };
 
-function createSoapBodyWithoutUrn({
+function SoapBody({
```

Comment:
nit: function name
- camelCase for function names
- use a verb to represent what the function does, not what it returns